python/tskit/trees.py

```diff
@@ -3254,6 +3256,47 @@ def draw_text(self, **kwargs):
 
     def general_stat(self, W, f, windows=None, polarised=False, mode=None,
                      span_normalise=True):
```

Comment:
Related to this: it may be nice to show the Python implementation of Tajima's D in the docs.  I think concrete examples are really helpful in making the generalities of the docstring less opaque.